src/components/login/SSOLogin.tsx

```diff
@@ -338,7 +410,10 @@ export default class SSOLogin extends Component<SSOLoginProps, SSOLoginState> {
                     lineDecorationsWidth={this.state.configMap === SwitchItemValues.Configuration ? 50 : 0}
                     shebang={shebangHtml}
                     readOnly={this.state.configMap !== SwitchItemValues.Configuration}
-                    onChange={(event) => { this.handleConfigChange(event) }}>
+                    onChange={(event) => { this.handleConfigChange(event) }}
```

Comment:
we can write directly onChange={this.handleConfigChange}
Same for other onFocus and onBlur